html-css/week1/nadiia/portfolio/index.html

```diff
@@ -0,0 +1,298 @@
+        <div class="cattail">
+          <div class="cattailmiddle"></div>
+          <div class="cattailstart"></div>
+          <div class="cattailclose"></div>
+          <div class="cattailend"></div>
+        </div>
+      </div>
+      <div class="welcome">
+        <p class="maintext">Hello, I'm Nadiia</p>
+        <p class="undertext">a designer, artist and web-developer</p>
+      </div>
+    </header>
+    <main>
+      <p class="art_project" id="about">About myself</p>
+      <p class="undertext plus">
+        I'm a person, who thinks about many things and shares them, providing my
+        ideas to web-sites or drawings. I like to express my feelings, emotions
+        and thoughts through canvas and it doesn't matter, if this canvas is on
+        a computer or on the table. If I have a free time, I prefer spending it
+        improving my skills.
+      </p>
+      <p class="art_project">Technical skills</p>
+      <p class="undertext plus">I'm mostly work with:</p>
+      <ul class="undertext plus">
+        <li>Figma - for designing web-sites!</li>
+        <li>Visual Studio Code - for creating web-sites!</li>
+        <li>Clip Studio Paint - for drawing and creating short animations!</li>
+        <li>Adobe Premiere Pro - for editing videos!</li>
+        <li>Adobe Photoshop - helps A LOT with creating animations.</li>
+      </ul>
+      <p class="undertext plus">
+        I know programming languages, such as HTML and CSS.
+      </p>
+      <p class="art_project" id="project">Projects</p>
+      <section id="project" class="project">
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225357803.png"
+            />
+            <p class="iframesundertext">Survey Form</p></a
+          >
+        </div>
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-tribute-page-project/build-a-tribute-page"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225519905.png"
+            />
+            <p class="iframesundertext">Tribute Page</p></a
+          >
+        </div>
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-technical-documentation-page-project/build-a-technical-documentation-page"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225640025.png"
+            />
+            <p class="iframesundertext">Technical Documentation Page</p></a
```

Comment:
I'd recommend to [use kebab case](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#heading-what-is-kebab-case) for better readability, unless you go for BEM.
```suggestion
            <span class="iframes-under-text">Technical Documentation Page</span></a
```

`<span>` tags can also be a bit better.

BEM name suggestion: `.projects__item-label`